docs/styles.css

```
@@ -0,0 +1,1107 @@
+  --color-heading: var(--color-navy);
+  /* Accent blue used AS TEXT (links, eyebrows, callout titles) -- distinct
+     from --color-accent-blue/-dark, which are also used as button/dot
+     BACKGROUNDS with white text and must not lighten (a lighter button
+     background would break the white-text-on-button contrast that already
+     works in both themes). Text usage needs a lighter blue on a dark
+     background; background usage doesn't. */
+  --color-link: var(--color-accent-blue);
+  --color-link-strong: var(--color-accent-blue-dark);
+
+  --font-sans:
+    ui-sans-serif, system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)